Watchman.Discord/Areas/Help/Services/HelpService.cs

```diff
@@ -17,8 +17,8 @@ namespace Watchman.Discord.Areas.Help.Services
     public interface IHelpService
     {
         IEnumerable<HelpInformation> GetHelpInformations(Contexts contexts);
-        Task PrintHelpForAllCommands(bool printAsJson, Contexts contexts, IEnumerable<HelpInformation> helpInformations);
-        Task PrintHelpForOneCommand(string commandName, Contexts contexts, IEnumerable<HelpInformation> helpInformations);
+        Task PrintHelpForAllCommands(bool isJson, Contexts contexts, IEnumerable<HelpInformation> helpInformations);
+        Task PrintHelpForOneCommand(string commandName, Contexts contexts, IEnumerable<HelpInformation> helpInformations, bool isJson = false);
```

Comment:
czemu w jednej metodzie isJson jest na początku a w drugiej na końcu?

+printAsJson bardziej mówi o co chodzi

Reply:
Gdyż w przypadku jednej komendy, json jest opcjonalny, dlatego jest na końcu.

Reply:
To w takim razie, isJson z linii 20 powinno też być na końcu.